controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -534,9 +532,32 @@ void wfa_ca::handle_wfa_ca_message(
             break;
         }
 
-        // Check if additional tlvs were added to command. If yes, then needs to parse the tlv and
-        // send it as it.
-        if (mandatory_params_num < params.size()) {
+        // Check if CMDU message type has preprepared CMDU which can be loaded
+        auto certification_tx_buffer = database.get_certification_tx_buffer();
+        if (!certification_tx_buffer) {
+            LOG(ERROR) << "certification_tx_buffer is not allocated";
+        }
+        ieee1905_1::CmduMessageTx external_cmdu_tx(
+            certification_tx_buffer.get() + sizeof(beerocks::message::sUdsHeader),
```

Comment:
possible segfault. 
you check for null before but didn't return. so possibly certification_tx_buffer is nullptr.